gpt4all-bindings/python/gpt4all/pyllmodel.py

```diff
@@ -369,11 +344,10 @@ def run_llmodel_prompt(model, prompt, prompt_callback, response_callback, recalc
     def _prompt_callback(token_id):
         return True
 
-    # Empty response callback method that just prints response to be collected
+
+    # Empty response callback
     @staticmethod
```

Comment:
Alright, one more thing because this is important:

You have a dangling `@staticmethod` here because you moved the code elsewehere. Needs to be removed.

(P.S. Hope we can chat about the PR and my review sometime soon.)